scripts/files/templatesV3.1.ts

```diff
@@ -58,27 +58,27 @@ export const templates: {
   11155111: {
     eth: {
       bridge: '0x331Edb39Eb823602484cD9F28bBC34cE4B9e76D9',
-      sequencerInbox: '0x9a4332b77d150AB19388e7F01B686De91a3Eadf1',
+      sequencerInbox: '0x79C5c3c8355daDC0d48D37eDC3Cb5ac73cB5b9d2',
```

Comment:
If this is how we set the templates ahead of time then we wouldn't even need to have somewhere to deploy the syndicate inbox changes. We don't have any changes in the inbox contract, so we could just point at some previously deployed version of their contract. Furthermore, even if this is how we want to pass implementation contracts to the future upgrades, we likely won't always need to update the inbox contract, or maintain a version of it because we don't have changes there.

I also want to point out that if this migration is establishing new transparent proxies for the rollup, we will be etching the current API's into the sequencer inbox proxy and will have to carry those forward forever, so it would be nice to take out anything we don't need to be there before this happens.

Reply:
Yes we can but we would need need to maintain a separate branch for syndicate I believe